src/App.vue

```diff
@@ -24,6 +24,8 @@
   var row2: number = getRandomNumbersInRange(1,0,24)
   var row3: number = getRandomNumbersInRange(1,0,24)
   var row4: number = getRandomNumbersInRange(1,0,24)
+  
+  document.getElementsByTagName( 'html' )[0].className = ' snes';
```

Comment:
I think I'd like the theme class to have a "theme-" prefix, just for clarity, and also just to make it so there could never be a class conflict from a theme name. Which yes, I know there are other approaches to that.

It's all about making me happy.